### DISCUSSION TITLE
Helper function

### DISCUSSION THREAD

**Discussion by Benjmanxd** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#37] [Q&A] [answered]
Is helper function allowed? Thanks.

**Comment by Derppening** [comment, marked as answer]
From [README](https://github.com/CastleLab/COMP3021-2021Fall-PA1-Student-Version#tasks):

> You may also add `private` methods to classes if it aids your implementation. Adding non-`private` methods are highly discouraged and may cause point deductions!

Some students did also ask whether they can add `public` helper classes (e.g. to reuse logic). I allow this, but these classes should only contain `static` methods and no instance variables. You should put instance variables in one of the provided skeleton files.
